Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
```

Comment:
Would there be value in using FROM perl:5   as a base image? Save on the apt installing

Reply:
note, i don't really worry overmuch about this, if anything it'll reduce image size and remove needing to set various envvars, but ...

Reply:
I did start with that but had problems getting it to work. Might have been a combination of SSL libraries and the env variables. The env variables are still (at least some of them) needed for things to work. Updated with perl:5 now :+1: